src/eko/evolution_operator/grid.py

```diff
@@ -111,7 +111,7 @@ def from_dict(
                 created object
         """
         config = {}
-        config["order"] = int(theory_card["PTO"])
+        config["order"] = theory_card["order"]
```

Comment:
Maybe make them explicitly integers as before (I don't remember if or which, but if `int` was there, there might have been a reason).
```suggestion
        config["order"] = tuple(int(o) for o in theory_card["order"])
```